.github/workflows/update-dotnet-install-script.yml

```diff
@@ -0,0 +1,48 @@
+name: "Updates vendor 'dotnet-install' script"
+on:
+  workflow_dispatch:
+  schedule:
+  - cron: '0 0 * * 0'  # Runs every Sunday at midnight UTC (adjust as needed)
+
+jobs:
+  fetch-latest-dotnet-install:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Run fetch-latest-dotnet-install.sh
+        run: src/dotnet/scripts/fetch-latest-dotnet-install.sh
+
+      - name: Create a PR for dotnet-install.sh
+        id: push_image_info
+        env:
+          GITHUB_TOKEN: ${{ secrets.PAT }}
+        run: |
+          set -e
+          echo "Start."
+
+          # Configure git and Push updates
+          git config --global user.email github-actions@github.com
+          git config --global user.name github-actions
+          git config pull.rebase false
+
+          branch=automated-script-update-$GITHUB_RUN_ID
+          git checkout -b $branch
+          message='[Updates] Automated vendor 'dotnet-install' script'
+
+          # Add / update and commit
+          git add src/dotnet/scripts/vendor/dotnet-install.sh
+          git commit -m 'Automated dotnet-install script update [skip ci]' || export NO_UPDATES=true
```

Comment:
@samruddhikhandale it just occurred to me - maybe this commit should go through all the automated validations? (i.e. Remove `[skip ci]`.) Unlikely, but an update to the vendor script could potentially break the dotnet feature.

Reply:
That's a very good point! I agree, let's remove `[skip ci]` bit?